DOCS-10131-[ION-8744] Double static IP for Cloudhub DLBs (#456)
mulesoft/docs-runtime-manager: modules/ROOT/pages/lb-updates.adoc

To update a load balancer:

. From Anypoint Platform, click *Runtime Manager*.
. Click *Load Balancers*.
. Click the *Update* button.
. At the *Update load balancer* prompt, click *Update*.
+
Click the *View release notes* link for details on the update. 
+

> [IMPORTANT]

Reviewer: Why is what follows in a lighter type?

====

> When updating a DLB with static IP addresses, the update might add more static IPs. 
> The update prompts you to add these these new IP addresses to your allowlist before continuing with the update.
> 
> In this case, add the IP addresses to your allowlist before clicking *Update*.

Reviewer: ```suggestion
When you update a DLB with static IP addresses, the update might require additional static IPs. If you are prompted to add these new IP addresses to your allowlist, do so before you click *Update*.
```
